your-turn/keisha.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="../main.css" rel="stylesheet">
+    <title>Keisha's Page</title>
+</head>
+<body>
+<h1>Keisha</h1>
+    <img src="https://mail.google.com/mail/u/0?ui=2&ik=c704e85f63&attid=0.1.1&permmsgid=msg-f:1757821459701026412&th=1865094de409ce6c&view=fimg&fur=ip&sz=s0-l75-ft&attbid=ANGjdJ8P5CRhJLT-81CP4JtJteS_kJvEKL00EifUxeLzyMLcp4ZtfYxhNGkYzpn5Ukv7jJ9JgGTyef2StSVI-aPcNA-WJWXKbpRv4xGMA73q0xjl7cBLgZjVfj2U0t4&disp=emb" class="your-image">
```

Comment:
Unfortunately, this link will not show up because it's local to your computer.
You can add another image by searching online for a picture and then right click and Copy Image Address and paste it

Reply:
I will try again. Thank you for the feedback.